packages/cli/test/integration/executions.controller.test.ts

```diff
@@ -1,19 +1,22 @@
+import type { User } from '@/databases/entities/User';
+import { createSuccessfulExecution, getAllExecutions } from './shared/db/executions';
+import { createOwner } from './shared/db/users';
+import { createWorkflow } from './shared/db/workflows';
 import * as testDb from './shared/testDb';
```

Comment:
Same here, only import `truncate` although I don't think it's a real problem since it's not re-exported